hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/HTSPrecompiledContract.java

```diff
@@ -238,14 +261,25 @@
 	//tokenURI(uint256 tokenId)
 	protected static final int ABI_ID_TOKEN_URI_NFT = 0xc87b56dd;
 
+	//createFungibleToken(HederaToken memory token, uint initialTotalSupply, uint decimals)
+	protected static final int ABI_ID_CREATE_FUNGIBLE_TOKEN = 0x7812a04b;
+	//createFungibleTokenWithCustomFees(HederaToken memory token, uint initialTotalSupply, uint decimals, FixedFee[] memory fixedFees, FractionalFee[] memory fractionalFees)
+	protected static final int ABI_ID_CREATE_FUNGIBLE_TOKEN_WITH_FEES = 0x4c381ae7;
+	//createNonFungibleToken(HederaToken memory token)
+	protected static final int ABI_ID_CREATE_NON_FUNGIBLE_TOKEN = 0x9dc711e0;
+	//createNonFungibleTokenWithCustomFees(HederaToken memory token, FixedFee[] memory fixedFees, RoyaltyFee[] memory royaltyFees)
+	protected static final int ABI_ID_CREATE_NON_FUNGIBLE_TOKEN_WITH_FEES = 0x5bc7c0e6;
+
 	//Transfer(address indexed from, address indexed to, uint256 indexed tokenId)
 	//Transfer(address indexed from, address indexed to, uint256 value)
 	private static final Bytes TRANSFER_EVENT = Bytes.fromHexString(
 			"ddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef");
 
+
 	private int functionId;
 	private Precompile precompile;
 	private TransactionBody.Builder transactionBody;
+	private MessageFrame messageFrame;
```

Comment:
Why are we tracking message frame?  If we need to it should have a better name, but in the one instance I see us use it we have a messageFrame as a parameter to the method.  If there is not a reason that can be used we should either add a comment there to clarify or use a name that makes that reason apparent. But I am skeptical we need to even track it in the precompile as it presents re-enterancy issues.

Reply:
Oops --- this is leftover from the initial implementation, where we used to set a specific `revertReason` of the frame when the `verifySolidityInput()` checks failed in `body()`. And since we do not have a reference to the frame in `body()` -> `messageFrame` member variable.

Anyway, looks like I've missed some stuff while changing to the current implementation --- thanks, Danno